client/src/views/Environment.tsx

```diff
@@ -73,6 +74,16 @@ function Environment() {
             </Button>
           }
 
+          <Button
+            variant="contained"
+            sx={{
+              color: theme => theme.palette.mode === 'dark' ? colors.brand.green.light : colors.brand.green.dark
```

Comment:
`color` is not changing the background, you should use `bgcolor` or `backgroundColor`. We need to also style the hover state of this button (this should be added in the theme, but for now it's ok to do it here). Here a suggestion, keeping the same button color for both light and dark themes:
```suggestion
              bgcolor: colors.brand.green.light,
              '&:hover': {
                filter: 'brightness(.9)',
                bgcolor: colors.brand.green.light,
              }
```